src/__generated__/ArtistForSaleArtworksGridQuery.graphql.ts

```diff
@@ -344,14 +344,20 @@ return {
                         "concreteType": "Artwork",
                         "plural": false,
                         "selections": [
+                          {
+                            "kind": "ScalarField",
+                            "alias": null,
+                            "name": "is_biddable",
+                            "args": null,
+                            "storageKey": null
+                          },
                           {
                             "kind": "ScalarField",
                             "alias": null,
                             "name": "id",
                             "args": null,
                             "storageKey": null
                           },
-                          v2,
```

Comment:
This looks similar to the churn seen in https://github.com/artsy/emission/pull/1227#discussion_r235180465, but as far as I can tell, no Relay queries were changed in this PR? /cc @orta @alloy

Reply:
Yes, only the order is different. I'll check if the relay compiler version changed

Reply:
It shouldn’t. This looks more like somebody has a plugin that is sorting keys, maybe?